docs/jwst/reset/description.rst

```diff
@@ -25,11 +25,9 @@ on the other hand, is the appearance of
 significant extra spatial structure in these initial
 groups, before fading out in later groups.
 
-The time constant associated with the reset anomaly is
-roughly a minute so for full array data the effect has faded out
-by ~group 20. On subarray data, where the read time  depends on
-the size of the subarray, the reset anomaly affects more
-groups in an integration.
+The reset anomaly effect fades out by ~group 15 for full array data. It takes a few more frames
```

Comment:
Mixed usage of "group" and "frame". Should pick one or the other and be consistent.